### PR TITLE
test: adding tests to acheive 100% coverage for blas/ext/base/dnansumkbn

### LINKED ISSUE #3046: Incomplete code coverage for `blas/ext/base/dnansumkbn`
See https://coverage.stdlib.io/blas/ext/base/dnansumkbn/index.html.

This should be straightforward to address. Just need to add a test case in each of the test files which handles the situation where the stride is `0` and the element is `NaN`.

### PR DESCRIPTION
**Pull request by Neerajpathak07**
Resolves #3046 

## Description

This pull request:

-   Adds a test case for every file to achieve 100% test coverage for blas/ext/base/dnansumkbn

## Related Issues

This pull request:

-   resolves #3046 

## Questions

No.

## Other

No.

## Checklist

-  [x] Read, understood, and followed the [contributing guidelines](https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md)

* * *

@stdlib-js/reviewers

### CODE REVIEW

**Comment by Neerajpathak07** [comment]
Was trying to re-run the test cases so closed this PR.

**Comment by headlessNode** [comment]
@Neerajpathak07 The title is misleading. You are adding tests to `dnannsumkbn` not `dnansumkbn`. Also, add opening and closing `backtick` around the package name in the title.

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.dnannsumkbn.js [hunk 1]
```diff
@@ -119,6 +119,17 @@ tape( 'the function calculates the sum of strided array elements (ignoring NaN v
...
+	var x;
+	var v;
+
+	x =new Float64Array([NaN , 1.0,2.0,3.0,4.0]);
+
+	v =dnannsumkbn(x.length, x ,0);
+	t.strictEqual(v,0.0, 'returns expected Value');
+	t.end();
+});
+
```
Comment (headlessNode) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.dnannsumkbn.js:132]:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', function test( t ) {
	var x;
	var v;

	x = new Float64Array( [ NaN, 1.0, 2.0, 3.0, 4.0 ] );

	v = dnannsumkbn( x.length, x, 0 );
	t.strictEqual( v, 0.0, 'returns expected Value' );
	t.end();
});
```

Comment (headlessNode) [reply lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.dnannsumkbn.js:132]: Make sure to adhere to the Style conventions. You can find them here: https://github.com/stdlib-js/stdlib/tree/develop/docs/style-guides

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.dnannsumkbn.native.js [hunk 2]
```diff
@@ -224,6 +224,17 @@ tape( 'the function supports negative `stride` parameters', opts, function test(
...
+	var x;
+	var v;
+
+	x =new Float64Array([NaN , 1.0,2.0,3.0,4.0]);
+
+	v =dnannsumkbn(x.length, x ,0);
+	t.strictEqual(v,0.0, 'returns expected Value');
+	t.end();
+});
+
```
Comment (headlessNode) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.dnannsumkbn.native.js:237]:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', opts, function test( t ) {
	var x;
	var v;

	x = new Float64Array( [ NaN, 1.0, 2.0, 3.0, 4.0 ] );

	v = dnannsumkbn( x.length, x, 0 );
	t.strictEqual( v, 0.0, 'returns expected Value' );
	t.end();
});

```

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.js [hunk 3]
```diff
@@ -63,6 +63,17 @@ tape( 'if a native implementation is available, the main export is the native im
...
+	var x;
+	var v;
+
+	x =new Float64Array([NaN , 1.0,2.0,3.0,4.0]);
+
+	v =dnannsumkbn(x.length, x ,0);
+	t.strictEqual(v,0.0, 'returns expected Value');
+	t.end();
+});
+
```
Comment (headlessNode) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.js:76]: No need to add it to the `test.js` file.
```suggestion
```

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.ndarray.js [hunk 4]
```diff
@@ -159,6 +159,17 @@ tape( 'if provided an `N` parameter equal to `1`, the function returns the first
...
+	var x;
+	var v;
+
+	x =new Float64Array([NaN , 1.0,2.0,3.0,4.0]);
+
+	v =dnannsumkbn(x.length, x ,0);
+	t.strictEqual(v,0.0, 'returns expected Value');
+	t.end();
+});
+
```
Comment (headlessNode) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.ndarray.js:172]:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', function test( t ) {
	var x;
	var v;

	x = new Float64Array( [ NaN, 1.0, 2.0, 3.0, 4.0 ] );

	v = dnannsumkbn( x.length, x, 0, 0 );
	t.strictEqual( v, 0.0, 'returns expected Value' );
	t.end();
});

```

File: lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.ndarray.native.js [hunk 5]
```diff
@@ -128,6 +128,17 @@ tape( 'the function calculates the sum of strided array elements (ignoring NaN v
...
+	var x;
+	var v;
+
+	x =new Float64Array([NaN , 1.0,2.0,3.0,4.0]);
+
+	v =dnannsumkbn(x.length, x ,0);
+	t.strictEqual(v,0.0, 'returns expected Value');
+	t.end();
+});
+
```
Comment (headlessNode) [inline lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.ndarray.native.js:141]:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', opts, function test( t ) {
	var x;
	var v;

	x = new Float64Array( [ NaN, 1.0, 2.0, 3.0, 4.0 ] );

	v = dnannsumkbn( x.length, x, 0, 0 );
	t.strictEqual( v, 0.0, 'returns expected Value' );
	t.end();
});

```

Comment (Neerajpathak07) [reply lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn/test/test.ndarray.native.js:141]: Ok, I will definitely look into the changes and create a pr again after changing the title.

**Pull request closed without merging**